src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs

```diff
@@ -283,6 +287,17 @@ We support
                         variants.Add(variant);
                     }
                 }
+
+                telemetryEnabled = configurationSection.GetValue<bool>("TelemetryEnabled");
```

Comment:
Should "TelemetryEnabled" be moved into ConfigurationFields.cs as a const string?